HelpSource/Classes/WebView.schelp

```diff
@@ -12,107 +12,296 @@ PRIVATE:: key
+
+METHOD:: onTitleChanged
+    Sets the object to be evaluated when the title of the displayed page changes.
+
+METHOD:: onUrlChanged
+    Sets the object to be evaluated when the url of the displayed page changes.
+
+METHOD:: onLoadProgress
+    Sets the object to be evaluated with load progress updates.
+
+METHOD:: onLoadStarted
+    Sets the object to be evaluated when a page load begins.
+
 METHOD:: onLoadFinished
     Sets the object to be evaluated when a page has loaded successfully, passing the view as the argument.
 
 METHOD:: onLoadFailed
     Sets the object to be evaluated when a page has failed to load, passing the view as the argument.
 
+METHOD:: overrideNavigation
+    When true, page navigation requests will not be handled. Use in combination with onLinkActivated to provide custom navigation behavior.
+
 METHOD:: onLinkActivated
-    Sets the object to be evaluated when the user triggers a link, passing the view and the URL of the link (as String) as the arguments.
+    Sets the object to be evaluated when the user triggers a link. Arguments are: a URL, a QWebPageNavigationType, a bool (page is opening in current view).
 
-    When this is set to other than nil, WebView will stop handling links altogether. Setting this to nil will restore WebView link handling again.
+    When this is set to other than nil, WebView will stop handling clicked links altogether. Setting this to nil will restore WebView link handling again.
+    Note: for specialty behavior when clicking on links, it is much better to use setUrlHandler in most cases.
 
-METHOD:: onReload
+METHOD:: onReloadTriggered
     Sets the object to be evaluated whenever a page reload is requested, passing the view and the URL to be reloaded (as String) as the arguments.
 
-    When this is set to other than nil, WebView will do nothing on reload requests. Setting this to nil will restore WebView reload handling again.
-
-
 SUBSECTION:: JavaScript
 
-METHOD:: evaluateJavaScript
+METHOD:: runJavaScript
     Evaluates the given JavaScript code in the context of the current page.
     Argument::
         A String.
+    Argument::
+        A function to be called when execution is complete, with the result as first argument.
+
+EXAMPLES::
+CODE::
+// A simple web browser
+(
+var browser, webview, reloadStop, prev, next, urlBox, go;
+
+webview 	= WebView()
+				.minSize_(300@200);
+reloadStop	= Button()
+				.states_([["※"], ["◙"]])
+				.fixedSize_(36@28);
+prev		= Button()
+				.states_([["⇦"]])
+				.fixedSize_(36@28);
+next		= Button()
+				.states_([["⇨"]])
+				.fixedSize_(36@28);
+urlBox		= TextField()
+				.minWidth_(100);
+go			= Button()
+				.states_([["⌘"]])
+				.fixedSize_(36@28);
+
+reloadStop.action = {
+	|v|
+	if (v.value == 1) {
+		webview.reload(true);
+	} {
+		webview.stop;
+	}
+};
+
+prev.action 	= { webview.back };
+next.action 	= { webview.forward };
+urlBox.action 	= { webview.url = urlBox.string };
+go.action 		= { webview.url = urlBox.string };
```

Comment:
tabs to align ='s here and above - this is not really style-consistent. would you mind changing it?